test/server/endpoints.js

```diff
@@ -224,7 +228,7 @@ server.init(9001, (error, server) => {
                             var latestNotification = JSON.parse(notifications[0]).eventID;
                             t.equal(latestNotification, 'event:400', 'A notification was created');
                             client.del('event:400');
-                            client.lpop('calendar:' + inviteeID);
+                            client.spop('calendar:' + inviteeID);
```

Comment:
So the order of items in a set isn't guaranteed - I ran into problems like this.  Better to use `srem` instead